holidays/holiday_base.py

```diff
@@ -483,6 +483,24 @@ def __getattr__(self, name):
                     + timedelta(days=+int(days) if delta_direction == "past" else -int(days)),
                 )
 
+            # Handle <n> days <past/prior> easter patterns (e.g.,
+            # _add_holiday_8_days_past_easter() or
+            # _add_holiday_5_days_prior_easter()).
+            nth_weekday_of_month_with_delta = re.match(
+                r"_add_holiday_(\d{1,2})_days_(past|prior)_easter",
```

Comment:
Let's keep it compatible with the rest:
```suggestion
                r"_add_holiday_(\d{1,2})_days?_(past|prior)_easter",
```

Reply:
Yes, it is used in not many countries (currently, 14). But pattern `"<past/prior> <last/<nth> <weekday> of <month>"` is used by 11. :)

Reply:
Let's place it as the last pattern.